pageAge.js

```diff
@@ -0,0 +1,21 @@
+import { Selector , t} from 'testcafe';
+
+  export default class AgePage{
+    constructor(){
+      this.userAge = '1996';
```

Comment:
it is better to store age in the constant in the test itself or in config file. because the age itself is not a part of the page, it is just test data